src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -533,6 +533,11 @@ public TimeZone getTimeZone() {
         return Strings.isNullOrEmpty(timezone) ? TimeZone.getTimeZone("UTC") : TimeZone.getTimeZone(timezone);
     }
 
+    public TimeZone getMessageTimeZone() {
+        String timezone = getString("secor.message.timezone");
```

Comment:
Should the variable be "secor.partition.timezone" so that it is more contextual?

Reply:
I was thinking of describing the configuration to denote the timezone in the events/messages. Do you think secor.partition.timezone will be more appropriate?